### ISSUE TITLE
Use 0.0.0.0 instead of 127.0.0.1

### ISSUE THREAD

**Issue by technosaurus** [dimkr/nss-block#1]
If a local web/proxy server is running, localhost could result in unwanted resolution.

**Comment by dimkr** [comment]
True, but if connect() fails because the address is malformed and a (well-written) application exists if connect() failed with anything but a certain, expected errno value (some corner case, say EINTR), nss-nss-block will break that application.
